File: API/Features/ShipRoutes/Dtos/ShipRouteReadDto.cs
namespace API.Features.ShipRoutes {

    public class ShipRouteReadDto {

        public int Id { get; set; }
        public string Description { get; set; }
        public string FromPort { get; set; }
        public string FromTime { get; set; }
        public string ViaPort { get; set; }
        public string ViaTime { get; set; }
        public string ToPort { get; set; }
        public string ToTime { get; set; }
        public bool IsActive { get; set; }
        public string User { get; set; }
        public string LastUpdate { get; set; }

    }

}

File: FrontEnd/src/app/features/shipRoutes/classes/dtos/shipRoute-read-dto.ts
import { Metadata } from 'src/app/shared/classes/metadata'

export interface ShipRouteReadDto extends Metadata {

    id: number
    description: string
    fromPort: string
    fromTime: string
    viaPort: string
    viaTime: string
    toPort: string
    toTime: string
    isActive: boolean

}
